tools/editor/plugins/canvas_item_editor_plugin.h

```diff
@@ -35,13 +35,19 @@
 #include "scene/gui/panel_container.h"
 #include "scene/gui/box_container.h"
 #include "scene/2d/canvas_item.h"
-
+#include "scene/2d/light_2d.h"
```

Comment:
Right. I thought I used it in `.h` cause I had worked in `.h` and move it all to `.cpp` later.
I will clean up those `#include`